packages/@pollyjs/core/src/utils/merge-configs.js

```diff
@@ -1,4 +1,32 @@
 import mergeWith from 'lodash-es/mergeWith';
+import { assert, EXPIRY_STRATEGIES } from '@pollyjs/utils';
+
+function deprecateRecordIfExpired(mergedConfig) {
+  const bothAreUsed =
+    mergedConfig.hasOwnProperty('recordIfExpired') &&
+    mergedConfig.hasOwnProperty('expiryStrategy');
```

Comment:
This will always be `true` since we merge the config with all the defaults.

Reply:
I believe the defaults were updated and the prop was removed, so this comment no longer applies.

Reply:
removed this check, it was happening every time we had both. For now, `recordIfExpired` just overrides `expiryStrategy`, but it logs a console.warn if you use it.